lib/types/src/receipt/envelope.rs

```diff
@@ -54,6 +54,9 @@ pub enum ZkReceiptEnvelope<T = Log, U = L2ToL1Log> {
     /// Receipt envelope with type flag 254, containing an upgrade transaction receipt.
     #[serde(rename = "0x7e")]
     Upgrade(ReceiptWithBloom<ZkReceipt<T, U>>),
+    /// Receipt envelope with type flag 125, containing an interop transaction receipt.
+    #[serde(rename = "0x7d")]
+    InteropRoots(ReceiptWithBloom<ZkReceipt<T, U>>),
```

Comment:
Also, do you know if there is any well-known tx type with id 125 already? I'd try to avoid collisions if possible.

Reply:
As far as I checked - no